ex02.ex

```diff
@@ -41,7 +41,7 @@ defmodule Ex02 do
   # Do the same using the & syntax #
   ##################################
 
-  list2b = your_anonymous_function(1, 2)
+  list2b = &([&1+&2, &1-&2])
```

Comment:
Just to note: the parens are not required here: `&[ &1+&2, ..]` works fine, and has less punctuation :)